rskj-core/src/main/java/co/rsk/rpc/Web3DebugModule.java

```diff
@@ -19,13 +19,16 @@
 package co.rsk.rpc;
 
 import co.rsk.rpc.modules.debug.DebugModule;
+import com.fasterxml.jackson.databind.JsonNode;
 
 public interface Web3DebugModule {
 
     default String debug_wireProtocolQueueSize() {
         return getDebugModule().wireProtocolQueueSize();
     }
 
+    default JsonNode debug_traceTransaction(String transactionHash) throws Exception { return getDebugModule().traceTransaction(transactionHash); };
```

Comment:
Please always start a new line after a `{`. We're about to add a linter to catch these kind of things!

Reply:
Done